wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -850,6 +855,174 @@ instance BuildableSafeGen Wallet where
 instance Buildable [Wallet] where
     build = bprint listJson
 
+
+
+--------------------------------------------------------------------------------
+-- Utxo statistics
+--------------------------------------------------------------------------------
+
+-- | Utxo statistics for the wallet.
+-- | Histogram is composed of bars that represent the bucket. The bucket is tagged by upper bound of a given bucket.
+-- | The bar value corresponds to the number of stakes
+-- | In the future the bar value could be different things:
+-- | (a) sum of stakes in a bucket
+-- | (b) avg or std of stake in a bucket
+-- | (c) topN buckets
+-- | to name a few
+newtype HistogramBar = HistogramBarCount (Text, Word64) deriving (Show, Eq, Ord, Generic)
+
+instance ToSchema HistogramBar where
+  declareNamedSchema = genericDeclareNamedSchema defaultSchemaOptions
+
+instance Arbitrary HistogramBar where
+  arbitrary =
+      let possibleBuckets = fmap show $ (zipWith (\ten toPower -> ten^toPower :: Word64) (repeat (10::Word64)) [(1::Word64)..16]) ++ [45 * (10^(15::Word64))]
```

Comment:
Again, we can re-use the `generateBounds` function here, right ?

Reply:
Reused